Sources/MuxStatsGoogleIMAPlugin/include/MuxImaListener+Private.h

```diff
@@ -1,13 +1,10 @@
 //
-//  Header.h
-//  
+//  MuxImaListener+Private.h
```

Comment:
Was this file intended to be deleted? Since we have `MUXSDKIMAAdsListener+Private.h`

Reply:
`MuxImaListener` is now deprecated as of this release. It's intended to be kept around to help transition to the new API for one or two releases. All the calls pass through to the new type. [See here](https://github.com/muxinc/mux-stats-google-ima/pull/44/files#diff-f34e962d1c12ce1a40baede891e9595ae975a59356c44e7cc29eda19317552b4).

Not strongly opposed to removing it, though.

Reply:
Ohh, I see now. Somehow I missed the annotation on `MuxImaListener.h`

Never mind, I'm good with keeping this